tests/components/vizio/test_config_flow.py

```diff
@@ -258,37 +236,15 @@ async def test_user_host_already_configured_no_port(
     )
 
     assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
-    assert result["errors"] == {CONF_HOST: "host_exists"}
+    assert result["errors"] == {CONF_HOST: "existing_config_entry_found"}
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)